docs/content/components/truncate.md

```diff
@@ -7,18 +7,52 @@ bundle: truncate
 ---
 
 
-`.css-truncate` will shorten text with an ellipsis. The maximum width of the truncated text can be changed by overriding the max-width of `.css-truncate-target`. Unless the full text is so long that it affects performace, always add `title` to the truncated element so the full text can still be seen.
+`.css-truncate` will shorten text with an ellipsis. Unless the full text is so long that it affects performace, always add `title` to the truncated element so the full text can still be seen.
```

Comment:
> My vote would be to also nix the "Unless the full text is so long that it affects performance," qualifier.

👍 Yes, let's remove that part. Hopefully there is some max character guard in code to prevent performance issues.